metadata-ingestion/setup.py

```diff
@@ -152,6 +152,27 @@ def get_long_description():
     },
     entry_points={
         "console_scripts": ["datahub = datahub.entrypoints:datahub"],
+        "datahub.ingestion.source": [
```

Comment:
I think the convention is to use underscores - e.g. Airflow uses `apache_airflow_provider` (see http://airflow.apache.org/docs/apache-airflow-providers/#how-to-create-your-own-provider)

Reply:
I saw the `.` convention was used in the python.org guide, but that might have just been an abbreviated example. I can switch to underscores we'd like to go with the Airflow convention.

https://packaging.python.org/guides/creating-and-discovering-plugins/#using-package-metadata
```python
setup(
    ...
    entry_points={'myapp.plugins': 'a = myapp_plugin_a'},
    ...
)
```

Reply:
Ah didn't catch that - let's leave it as is then

Reply:
Are you good with this key? 

We can also change it to something like `datahub.ingestion.{source,sink}.plugins` if you like that better.